src/views/design/ScriptDetail/index.tsx

```diff
@@ -599,7 +599,7 @@ const ScriptDetail = withStyles(styles)(
             return (
                 <EditAction
                     onClose={() => this.setState({ editActionIndex: -1 })}
-                    action={this.getEditAction()}
+                    action={clone(this.getEditAction())}
```

Comment:
is `this.getEditAction()` used anywhere else? Maybe this method should return the clone?

Reply:
yep, it is mainly used in EditAction component, and also in the variable editAction, but I think it has no impact on the code, I just checked it out, and nothing strange appear